lib/alephant/lookup.rb

```diff
@@ -9,9 +9,9 @@ module Lookup
     include Logger
     @@lookup_tables = {}
 
-    def self.create(table_name)
+    def self.create(table_name, config={})
```

Comment:
@samfrench I think the above is fine. I'm not really a big fan of method signatures that have a single argument which is just a hash of stuff. I like the idea that the method has a baseline requirement of one argument (the table name to be created) and then anything beyond that is configurable by passing in data via a config object

So I'd suggest leaving the above 'as is'. 

Reading in config from the Broker would be tightly coupling this gem to the broker and would cause other problems in the future I fear. So best to inject rather than couple the code